src/waveform/renderers/waveformwidgetrenderer.cpp

```diff
@@ -242,6 +225,76 @@ void WaveformWidgetRenderer::draw(QPainter* painter, QPaintEvent* event) {
     //qDebug() << "draw() end" << timer.restart().formatNanosWithUnit();
 }
 
+void WaveformWidgetRenderer::drawPlayPosmarker(QPainter* painter) {
+    const int lineX = m_width * m_playMarkerPosition;
+    const int lineY = m_height * m_playMarkerPosition;
+
+    // draw dim outlines to increase playpos/waveform contrast
+    painter->setOpacity(0.5);
+    painter->setPen(m_colors.getBgColor());
+    if (m_orientation == Qt::Horizontal) {
+        // lines next to playpos
+        // Note: don't draw lines where they would overlap the triangles,
+        // otherwise the translucent strokes add up to a darker tone.
+        painter->drawLine(lineX + 1, 4, lineX + 1, m_height - 3);
+        painter->drawLine(lineX - 1, 4, lineX - 1, m_height - 3);
+
+        // Non-rectangular lines are less than 1px wide, so let's draw the
+        // triangles more opaque to have same effective line/waveform contrast.
+        painter->setOpacity(0.8);
+        // top triangle
+        drawTriangle(painter, m_colors.getBgColor(),
```

Comment:
This is not a real issue here, but this is a line break clang-format rejects. 
It allows either a single line of parameters or one parameter at each line.

Reply:
okay.
But I'd like to have the coordinates easily readable, so I'll set each QPoint individually and call `drawTriangle(painter, m_colors.getBgColor(), p0, p1, p2);`
Any objections?

Reply:
Perfect :-)